opensearch_dashboards.json

```diff
@@ -20,5 +20,7 @@
     "opensearchDashboardsUtils"
   ],
   "server": true,
-  "ui": true
+  "ui": true,
+  "supportedOSDataSourceVersions": ">=2.15.0",
```

Comment:
As we discussed, this doesn't have to be 2.15. We may start with feature anywhere release.

Reply:
updated it to 2.9, the version we launched feature anywhere